apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -293,6 +293,8 @@ type EnvoyConfig struct {
 	// Network holds various configurable Envoy network values.
 	// +optional
 	Network *NetworkParameters `json:"network,omitempty"`
+
+	Tracing *TracingConfig `json:"tracing,omitempty"`
```

Comment:
I think it probably makes sense for this to be a top-level field on the spec, i.e. as a sibling of `RateLimitService`.

Reply:
ok